integration-cli/docker_cli_commit_test.go

```diff
@@ -63,11 +63,8 @@ func (s *DockerSuite) TestCommitNewFile(c *check.C) {
 	imageID = strings.Trim(imageID, "\r\n")
 
 	out, _ := dockerCmd(c, "run", imageID, "cat", "/foo")
-
-	if actual := strings.Trim(out, "\r\n"); actual != "koye" {
-		c.Fatalf("expected output koye received %q", actual)
-	}
-
+	actual := strings.Trim(out, "\r\n")
```

Comment:
Done - I went and got rid of all instances of strings.Trim(), and replaced it with strings.TrimSpace()